backend-api/tests/infra-tests/application.test.ts

```diff
@@ -244,7 +244,7 @@ describe("Backend application infrastructure", () => {
     });
 
     test("All alarms are configured with a Condition", () => {
-      const conditionalNames = ["DeployAlarms", "UseLinearCanaryDeployments"];
+      const conditionalNames = ["DeployAlarms", "UseLambdaCanaryDeployments"];
```

Comment:
Todo: remove ununsed condition 
```suggestion
      const conditionalNames = ["DeployAlarms"];
```